EventSelectionEfficiency/20240922_evtSelEffCorr/evtSelEffCorr.cpp

```diff
@@ -97,21 +96,23 @@ int main(int argc, char *argv[])
     
    }
 
-   EnergyBins[0] = 0;
-   for(int e = 1; e <= EnergyBinCount; e++){
-      double logValue = logMin + e * logStep;
-      EnergyBins[e] =  std::pow(10, logValue);
-   }
+   // EnergyBins[0] = 0;
+   // for(int e = 1; e <= EnergyBinCount; e++){
+   //    double logValue = logMin + e * logStep;
+   //    EnergyBins[e] =  std::pow(10, logValue);
+   //    printf("%f ", EnergyBins[e]);
+   // }
```

Comment:
Let’s keep in mind to make the bin boundary configurable in the future.  How about we add a comment so in the future we know to come back to update?